.github/workflows/update-targets.yml

```
@@ -61,6 +62,7 @@ jobs:
           delete-branch: true
 
       - name: Enable Pull Request Automerge
+        if: ${{ steps.cpr.outputs.pull-request-number }}
         run: gh pr merge --merge --auto "1"
```

we could also pass here the PR number as argument

https://cli.github.com/manual/gh_pr_merge

---

`gh pr merge [<number> | <url> | <branch>] [flags]`

something like this
```suggestion
        run: gh pr merge ${{ steps.cpr.outputs.pull-request-number }} --merge --auto "1"
```

or
`${{ steps.cpr.outputs.pull-request-url }}`